package/main/daml/ContingentClaims.Lifecycle/daml.yaml

```diff
@@ -11,5 +11,5 @@ dependencies:
 data-dependencies:
   - .lib/daml-ctl/v2.3.0/daml-ctl-2.3.0.dar
   - .lib/daml-finance/ContingentClaims.Core/2.0.0/contingent-claims-core-2.0.0.dar
-build-options: null
```

Comment:
In general I would put these stylistic changes in a separate PR

Reply:
makes sense, will revert it

Reply:
no need, but in general it's good practice I believe :)